packages/turf-projection/index.ts

```diff
@@ -82,25 +82,32 @@ function convert(geojson: any, projection: string, options: {mutate?: boolean} =
  *
  * @private
  * @param {Array<number>} lonLat WGS84 point
+ * @param {Object} [options] Optional parameters
+ * @param {boolean} [options.wrapLongitude=true] wrap logitude passing the meridian to a coordinated between -180 and 180
  * @returns {Array<number>} Mercator [x, y] point
  */
-function convertToMercator(lonLat) {
+function convertToMercator(lonLat, options: {wrapLongitude?: boolean} = {wrapLongitude: true}) {
```

Comment:
I'm not sure I understand why you added this optional parameter if the function is only used internally (it's not exported).
besides, I was under the impression wrapping the coordinates was actually the original issue, so why keeping the option... 🤔

Reply:
In my opinion even it polygons must be allowed to overlap the antimeridian / meridian, for points, I think we would prefer them to always be wrapped. 
So I set the wrapLongitude to true only for geojson, if someone sends a `[lng, lat]` array we keep the longitude wrapping as it would be the behavior i would be expecting.

What do you think ?

Reply:
I'm not sure I can express the same thing with a optional parameter on the `toMercator` / `toWgs84` functions. 

Both method takes either coordinates array or geojson as parameter, the default value we want is different depending on the nature of the parameter type, would how would you name such parameter ?

Reply:
I don't think we want a different default value, and so behavior, depending solely on the type of input, rather just giving to the user the option to decide.

I believe the expected result in Turf would be not having wrapped coords (basically allow exceeding the -180/180 range). That's why the [original issue](#1602) is considered a bug.

Reply:
BTW the parameter `options: {wrapLongitude?: boolean} ` I think is very appropriate also for those functions

Reply:
ok then, i'll add the option, and the tests